cs/Openfin.PlatformAPI.Demo/layout.json

```diff
@@ -0,0 +1,21 @@
+{
+    "layout": {
+        "content": [
+            {
+                "type": "stack",
+                "id": "no-drop-target",
+                "content": [
+                    {
+                        "type": "component",
+                        "componentName": "view",
+                        "componentState": {
+                            "name": "%name%",
```

Comment:
Name hot needed, but we would not need this file if we are only storing data in memory.